src/lib/chat/matrix/chat-message.ts

```diff
@@ -46,9 +48,35 @@ export async function mapMatrixMessage(matrixMessage, sdkMatrixClient: SDKMatrix
     },
     isAdmin: false,
     optimisticId: content.optimisticId,
-    ...{ mentionedUsers: [], hidePreview: false, media: null, image: null, admin: {} },
+    ...{
+      admin: {},
+      mentionedUsers: [],
+      hidePreview: false,
+      media: null,
+      image: null,
+    },
```

Comment:
I don't understand why this is "splatting" an object that we're just creating. Why don't we just put these properties on the object directly?

Reply:
This was existing but I will update in the follow up PRs, no reason that I can see for the 'splatting'